main/src/ca/uwaterloo/flix/language/errors/ResolutionError.scala

```diff
@@ -1083,8 +1083,8 @@ object ResolutionError {
     * @param loc      the location where the error occurred.
     */
   case class MismatchedOpArity(op: Symbol.OpSym, expected: Int, actual: Int, loc: SourceLocation) extends ResolutionError with Recoverable {
-    val params = if (expected == 1) "parameter" else "parameters"
-    val be = if (actual == 1) "is" else "are"
+    val params: String = if (expected == 1) "parameter" else "parameters"
+    val be: String = if (actual == 1) "is" else "are"
```

Comment:
We have something called `Grammar.scala`. Could this be moved there?

Reply:
I did not write it :)